frontend/src/components/maps/leaflet/PointClusterer.tsx

```diff
@@ -52,9 +53,38 @@ export const convertToProperty = (
   longitude?: number,
 ): IProperty | null => {
   if ([PropertyTypes.Land, PropertyTypes.Subdivision].includes(property.propertyTypeId)) {
```

Comment:
Wonder if the PointClusterer should be generic and the conversion of a property should be based on a separate abstraction.

Reply:
Agree. Prob a tech-debt ticket

Reply:
Yeah, there are a number of tech-debt issues that I've logged with this ticket.